posts/jhipster-5-release.md

```diff
@@ -0,0 +1,218 @@
+---
+authors:
+- Kenneth Hegeland, Theo Lebrun, Raphael Brugier
+tags:
+- Brace yourself, JHipster 5 is out!
```

Comment:
This looks like a title. Tags should be one word, e.g. JHipster.

Reply:
👍